8/windows/nanoserver-1809/Dockerfile

```diff
@@ -55,8 +55,6 @@ ARG JENKINS_AGENT_WORK="C:/Users/${user}/Work"
 ENV JENKINS_AGENT_USER ${user}
 ENV JENKINS_AGENT_WORK ${JENKINS_AGENT_WORK}
 
-USER ContainerAdministrator

```

Comment:
This change related?

Reply:
I should have listed that in the PR, it's a redundant `USER` directive